Gemfile.lock

```diff
@@ -5,30 +5,30 @@ PATH
       activesupport (>= 3.2.15, < 4)
       clamp (~> 0.6.3)
       colored (~> 1.2)
-      xcodeproj (~> 0.18)
+      xcodeproj (>= 0.28.2, < 1.1.0)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (3.2.22)
+    activesupport (3.2.22.1)
       i18n (~> 0.6, >= 0.6.4)
       multi_json (~> 1.0)
-    ast (2.0.0)
+    ast (2.2.0)
     bacon (1.2.0)
     claide (0.9.1)
     clamp (0.6.5)
-    clintegracon (0.5.2)
+    clintegracon (0.8.1)
```

Comment:
@pietbrauer: This comes likely through the version bump here. I'll take care of that.